win720.py

```
@@ -52,69 +54,110 @@ def buy_Win720(
+        try:
+            res = self.http_client.get(
+                "https://dhlottery.co.kr/common.do?method=main",
+                headers=self._REQ_HEADERS
+            )
+            html = res.text
+            soup = BS(html, "html5lib")
+            found = soup.find("strong", id="drwNo720")
+            if found:
+                return str(int(found.text) - 1)
+            else:
+                raise ValueError("drwNo720 not found")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)